ballerina/tests/numerical-query-test.bal

```diff
@@ -296,6 +296,25 @@ function testQueryFromNullTable() returns error? {
     test:assertEquals(returnData["REAL_TYPE"], ());
 }
 
+type DataTable record {
+    int int_type;
+};
+
+@test:Config {
+    groups: ["query", "query-numeric-params"]
+}
+function testQueryDatabaseError() returns error? {
+    MockClient dbClient = check new (url = jdbcURL, user = user, password = password);
+    stream<DataTable, Error?> streamData =
+            <stream<DataTable, Error?>> dbClient->query(`SELECT int_type from DataTable1`, DataTable);
```

Comment:
```suggestion
    stream<DataTable, Error?> streamData = dbClient->query(`SELECT int_type from DataTable1`);
```

Reply:
This is intentional. This error occurs only with this type of casting before beta2. Else this would be handled at the stream level.

Reply:
Ack